mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
```

Comment:
Here is the review:

In the `exports.deleteOrder` function, the `try` block is not checking if `req.params.orderId` is valid before calling `Order.findByIdAndRemove`. This could lead to errors if `req.params.orderId` is null or undefined. 

Additionally, the `catch` block is catching all errors, but it would be better to catch specific errors that can occur during the deletion process, such as `MongoError` or `CastError`. This would allow for more targeted error handling.